e2etests/tests/evpn_l2.go

```diff
@@ -103,6 +121,12 @@ var _ = Describe("Routes between bgp and the fabric", Ordered, func() {
 		Expect(err).NotTo(HaveOccurred())
 		l2VniRedWithGateway := l2VniRed.DeepCopy()
 		l2VniRedWithGateway.Spec.L2GatewayIPs = tc.l2GatewayIPs
+
+		// Use hostMaster from test case if provided
+		if tc.hostMaster != nil {
+			l2VniRedWithGateway.Spec.HostMaster = tc.hostMaster
```

Comment:
let's not rely on defaults and be explicit by passing host master from outside

Reply:
Sure. Makes sense, good catch.